launcher/CMakeLists.txt

```diff
@@ -568,6 +568,23 @@ set(LOGIC_SOURCES
     ${ATLAUNCHER_SOURCES}
 )
 
+ecm_qt_declare_logging_category(LOGIC_SOURCES
+    HEADER prismlauncherdebug.h
+    IDENTIFIER PRISM_LAUNCHER
```

Comment:
This identifier should be defined as a variable in `program_info/CMakeLists.txt` and referenced here